resources/assets/js/label-trees/components/labelTreeLabel.vue

```diff
@@ -13,6 +13,10 @@
             </span>
             <span class="label-tree-label__buttons">
                 <button v-if="showFavourites" type="button" class="label-tree-label__favourite" :class="favouriteClass" @click.stop="toggleFavourite" :title="favouriteTitle">
+                    <span v-if="showFavouriteShortcuts" @click.stop="!toggleFavourite" >
```

Comment:
Can you please add a CSS class to the `<span>` and define `pointer-events: none;` for it (in [this file](https://github.com/biigle/core/blob/master/resources/assets/sass/label-trees/components/_labelTreeLabel.scss))? With this you can click on the icon and number and the label is still de-/selected.

Reply:
Also you could ass the `text-muted` class to make the icon and number a little darker. Then there is a better distinction between the icon/number and the star icon.